ray-on-gke/tpu/kuberay-tpu-webhook/deployments/deployment.yaml

```diff
@@ -3,6 +3,28 @@ kind: Namespace
 metadata:
   name: ray-system
 ---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: pod-reader
```

Comment:
ClusterRole name should be more specific, `kuberay-tpu-webhook-pod-reader`

Reply:
[6f1fd5d](https://github.com/GoogleCloudPlatform/ai-on-gke/pull/723/commits/6f1fd5d7c85c5b9860d077a8e233feb825b05edd)